### PR TITLE
Update Jenkinsfile

### PR DESCRIPTION
**Pull request by tmbgreaves**
Trivial change to test PR integration

### CODE REVIEW

File: Jenkinsfile [hunk 1]
```diff
@@ -6,7 +6,7 @@ pipeline {
     PATH = "/usr/local/bin:/usr/bin:/bin"
   }
   stages {
-    stage('True') {
```
Comment (FabioLuporini) [inline Jenkinsfile]: are you dropping a (or keeping an extra)  ``'`` ?

**Comment by tmbgreaves** [comment]
I think that once Jenkins is up and working correctly the change-over is fairly easy - when everyone is happy Jenkins is doing the right thing, turn off travis.

In the meantime this branch (and associated PRs) are aimed at testing and attempting to break the Jenkins setup so can probably be safely ignored :-)

**Comment by ggorman** [comment]
@tmbgreaves - Can this be rebased and verify that it's working?
@FabioLuporini - Decision time, flip?

**Comment by FabioLuporini** [comment]
> @FabioLuporini - Decision time, flip?

unless I'm missing something obvious, so far this is only killing our ``.travis.yml``, while the current ``jenkinsfile`` (not sure what's the standard name convention for it) is basically doing nothing

**Comment by tmbgreaves** [comment]
Yes - I'd dropped the travis yaml so hacking on this branch didn't trigger lots of travis builds, but haven't yet done the porting over of travis setup to Jenkins. This Jenkinsfile is a placeholder checking that Jenkins finds and does a minimal run on a given branch/pr update. I'll aim to get the Jenkinsfile working this week at which point I'd suggest giving it a couple of weeks minimum running both CIs and checking they agree on passes/fails before switching.

**Pull request merged**